internal/app/networkd/pkg/nic/nic.go

```diff
@@ -401,7 +409,10 @@ func (n *NetworkInterface) configureInterface(method address.Addressing, link *n
 
 		err = n.rtnlConn.RouteAdd(method.Link(), *r.Destination, gw, rtnl.WithRouteSrc(src), rtnl.WithRouteAttrs(attr))
 		if err != nil {
-			return err
+			// ignore "EEXIST" errors for routes which are already present
```

Comment:
this is the "core" of the fix as if routes don't change, trying to add them always returns `file exists` error